vispy/plot/plotwidget.py

```diff
@@ -162,27 +162,36 @@ def _configure_3d(self):
 
         self._configured = True
 
-    def histogram(self, data, bins=10, color='w', orientation='h'):
+    def histogram(self, data=None, bins=10, color='w', orientation='h', **kwargs):
         """Calculate and show a histogram of data
 
         Parameters
         ----------
-        data : array-like
-            Data to histogram. Currently only 1D data is supported.
-        bins : int | array-like
-            Number of bins, or bin edges.
+        data : array-like, optional
+            Data to histogram.  May be `None` on initialization, use `set_raw_data`
+            on returned object to set data after initialization.
+        bins : int | array-like | str
+            If `bins` is an int, it defines the number of equal-width
+            bins in the given range (10, by default). If `bins` is a
+            sequence, it defines a monotonically increasing array of bin edges,
+            including the rightmost edge, allowing for non-uniform bin widths.
+            May also be a string if the calc_hist function supports it.
```

Comment:
Should this, instead of being detailed here, just point to the HistogramVisual object?